node-gui/src/main_window/main_widget/tabs/wallet/addresses.rs

```diff
@@ -23,6 +23,8 @@ use crate::backend::messages::AccountInfo;
 
 use super::WalletMessage;
 
+const NEW_ADDRESS_TOOLTIP_TEXT: &str = "TODO";
```

Comment:
Can we put something else here? Even if it's just something like "Generate a new address".

Reply:
Please come back in 15 minutes, you'll see all the texts. I do need scrutiny once I'm done.